developer-docs/installation/install_mobile_setup.md

```diff
@@ -68,10 +68,10 @@ Instance admin of Sunbird adopters can configure various aspects of the Sunbird
 
 | S No. | Variable Name | Description | Purpose | Default Value
 |-------|---------------|--------------|--------|--------------
-| 1 | appId | the app id in sunbird-mobile/config.xml with implementation specific application ID | To change the app ID | appId: "org.sunbird.app"
-| 2 | app name | navigate to sunbird-mobile/config.xml and enter the required app name | To change the app name | 
-| 3 | app logo | navigate to sunbird-mobile/resources/android/icon and sunbird-mobile/resources/android/splash. In all the mipmap and drawable folders, replace ic_launcher.png image with your desired logo. The logo name should exactly match the text **drawable-ldpi-icon.png** | To change the app logo |
-| 4 | app | Set the configuration variable inside the <b>sunbird-mobile repo</b> inside <b>buildConfig</b> folder | | 
+| 1 | dev_app_id/staging_app_id/production_app_id | The app ID in the **sunbird-mobile/buildConfig/sunbird.properties** file with the implementation-specific application ID | To change the app ID | appId: "org.sunbird.app"
+| 2 | app name | Navigate to the **sunbird-mobile/config.xml** file and enter the required app name. | To change the app name | 
+| 3 | app logo | Navigate to **sunbird-mobile/resources/android/icon** and **sunbird-mobile/resources/android/splash**. Replace the **ic_launcher.png** image with your desired logo in all the mipmap and drawable folders. The logo name should exactly match the text **drawable-ldpi-icon.png** | To change the app logo |

```

Comment:
@swayangjit please clarify which text should match the logo name. Default values are not provided for most of the variables. please provide the same.

Reply:
@Sunitashelar  Whatever image adaptor will put that image file name should be same as **drawable-ldpi-icon.png**.